sources/packages/backend/apps/db-migrations/src/sql/Reports/Update-daily-disbursement-file-zero-records.sql

```diff
@@ -0,0 +1,128 @@
+                    ) AS "Full Time BC Student Grant",
+                    SUM(
+                        full_time_disbursement_receipts.total_disbursed_amount +
+                        full_time_disbursement_receipts.total_disbursed_grant_amount
+                    ) AS "Full Time BC Total",
+                    0 AS "Part Time BC Student Grant",
+                    0 AS "Part Time BC Total",
+                    SUM(
+                        full_time_disbursement_receipts.total_disbursed_amount +
+                        full_time_disbursement_receipts.total_disbursed_grant_amount
+                    ) AS "BC Total",
+                    COUNT(full_time_disbursement_receipts.id) AS "Total Records",
+                    full_time_disbursement_receipts.file_date AS "File Date",
+                    full_time_disbursement_receipts.batch_run_date AS "Batch Run Date",
+                    full_time_disbursement_receipts.sequence_number AS "Sequence Number"
+                FROM
+                    disbursement_receipt_dataset full_time_disbursement_receipts
+                WHERE
+                    full_time_disbursement_receipts.funding_type = ''BC''
+                GROUP BY
+                    "File Date",
+                    "Batch Run Date",
+                    "Sequence Number"
+                UNION
+                ALL
+                SELECT
+                    0 AS "Full Time BC Student Loan",
+                    0 AS "Full Time BC Student Grant",
+                    0 AS "Full Time BC Total",
+                    SUM(
+                        part_time_disbursement_receipts.total_disbursed_grant_amount
+                    ) AS "Part Time BC Student Grant",
+                    SUM(
+                        part_time_disbursement_receipts.total_disbursed_grant_amount
+                    ) AS "Part Time BC Total",
+                    SUM(
+                        part_time_disbursement_receipts.total_disbursed_grant_amount
+                    ) AS "BC Total",
+                    COUNT(part_time_disbursement_receipts.id) AS "Total Records",
+                    part_time_disbursement_receipts.file_date as "File Date",
+                    part_time_disbursement_receipts.batch_run_date as "Batch Run Date",
+                    part_time_disbursement_receipts.sequence_number as "Sequence Number"
+                FROM
+                    disbursement_receipt_dataset part_time_disbursement_receipts
+                WHERE
+                    part_time_disbursement_receipts.funding_type = ''BP''
+                GROUP BY
+                    "File Date",
+                    "Batch Run Date",
+                    "Sequence Number"
+            ) AS disbursement_receipts
+        GROUP BY
+            disbursement_receipts."File Date",
+            disbursement_receipts."Batch Run Date",
+            disbursement_receipts."Sequence Number"
+        UNION
+        ALL 
```

Comment:
While setting a default record on result set being empty UNION vs UNION ALL does not make any difference and UNION is more accurate here. But I won't consider it as blocker.